clustered/integration-test/src/test/java/org/ehcache/clustered/management/ClusteringManagementServiceTest.java

```diff
@@ -48,99 +41,57 @@
-  private static final Collection<Descriptor> CACHE_DESCRIPTORS = new ArrayList<Descriptor>();
-
-  private static AtomicInteger N = new AtomicInteger();
-
-  @Rule
-  public final Timeout globalTimeout = Timeout.seconds(60);
-
-  private CacheManager cacheManager;
-  private String clientIdentifier;
-  private int n = N.incrementAndGet();
-
-  @Before
-  public void init() throws Exception {
-    // clear previous messages
-    clear();
-
-    this.cacheManager = CacheManagerBuilder.newCacheManagerBuilder()
-      // cluster config
-      .with(ClusteringServiceConfigurationBuilder.cluster(CLUSTER.getConnectionURI().resolve("/my-server-entity-" + n))
-        .autoCreate()
-        .defaultServerResource("primary-server-resource"))
-      // management config
-      .using(new DefaultManagementRegistryConfiguration()
-        .addTags("webapp-1", "server-node-1")
-        .setCacheManagerAlias("my-super-cache-manager")
-        .addConfiguration(new EhcacheStatisticsProviderConfiguration(
-          1, TimeUnit.MINUTES,
-          100, 1, TimeUnit.SECONDS,
-          2, TimeUnit.SECONDS))) // TTD reduce to 2 seconds so that the stat collector runs faster
-      // cache config
-      .withCache("cache-1", CacheConfigurationBuilder.newCacheConfigurationBuilder(
-        String.class, String.class,
-        newResourcePoolsBuilder()
-          .heap(10, EntryUnit.ENTRIES)
-          .offheap(1, MemoryUnit.MB)
-          .with(ClusteredResourcePoolBuilder.clusteredDedicated("primary-server-resource", 2, MemoryUnit.MB)))
-        .build())
-      .build(true);
-
-    // ensure the CM is running and get its client id
-    assertThat(cacheManager.getStatus(), equalTo(Status.AVAILABLE));
-    for (Client client : consumer.readTopology().getClients().values()) {
-      if(client.getName().equals("Ehcache:my-server-entity-" + n)) {
-        clientIdentifier = client.getClientId();
-      }
-    }
-    assertThat(clientIdentifier, is(notNullValue()));
-  }
+  private static final Collection<Descriptor> ONHEAP_DESCRIPTORS = new ArrayList<>();
+  private static final Collection<Descriptor> OFFHEAP_DESCRIPTORS = new ArrayList<>();
+  private static final Collection<Descriptor> DISK_DESCRIPTORS =  new ArrayList<>();
+  private static final Collection<Descriptor> CLUSTERED_DESCRIPTORS =  new ArrayList<>();
+  private static final Collection<Descriptor> CACHE_DESCRIPTORS = new ArrayList<>();
 
-  @After
-  public void close() throws Exception {
-    if (cacheManager != null && cacheManager.getStatus() == Status.AVAILABLE) {
-      cacheManager.close();
-    }
+  @Test
+  @Ignore("This is not a test, but something useful to show a json print of a cluster topology with all management metadata inside")
+  public void test_A_topology() throws Exception {
+    Cluster cluster = consumer.readTopology();
+    String json = mapper.writeValueAsString(cluster.toMap());
+    System.out.println(json);
+    assertThat(json, equalTo(read("/topology.json")));
```

Comment:
I missed the @Ignore indeed and the comment it contains. Fine for me to leave that in, just get rid of the assertion 😉